packages/lesmis/src/components/Chart.js

```diff
@@ -72,21 +74,24 @@ const Toggle = ({ value, onChange }) => (
 );
 
 // eslint-disable-next-line react/prop-types
-const ChartTable = ({ viewContent, isLoading, isError, error, selectedTab }) => (
-  <FetchLoader isLoading={isLoading} isError={isError} error={error}>
-    {selectedTab === TABS.CHART ? (
-      <ChartWrapper>
-        <ChartComponent viewContent={viewContent} legendPosition="top" />
-      </ChartWrapper>
-    ) : (
-      <Wrapper>
-        <Table viewContent={viewContent} />
-      </Wrapper>
-    )}
-  </FetchLoader>
-);
+const ChartTable = ({ viewContent, isLoading, isFetching, isError, error, selectedTab }) => {
+  const showIsLoading = isLoading || (!getIsChartData(viewContent) && isFetching);
```

Comment:
Also show the full loader if there is no data and we are fetching new data in the backgrond

Reply:
Nice touch, so we keep the animations is there is already data but show a more obvious loader when there is no data 👌